vendor/ddcloud/resource_additional_nics.go

```diff
@@ -0,0 +1,75 @@
+package ddcloud
+
+import "github.com/hashicorp/terraform/helper/schema"
+
+const (
+	resourceKeyNicServerID     = "serverid"
+	resourceKeyNicID           = "id"
+	resourceKeyNicVLANID       = "vlan_id"
+	resourceKeyNicPriavateIPV4 = "private_ipv4"
+	resourceKeyNicPrivateIPV6  = "private_ipv6"
+	resourceKeyIsShutdownOk    = "shutdown_ok"
+)
+
+func resourceAdditionalNic() *schema.Resource {
+	return &schema.Resource{
```

Comment:
Can all of these properties be changed after creation, or do any require a delete-and-recreate?